AmplifyPlugins/API/Sources/AWSAPIPlugin/Interceptor/RequestInterceptor/IAMURLRequestInterceptor.swift

```diff
@@ -36,7 +36,6 @@ struct IAMURLRequestInterceptor: URLRequestInterceptor {
             throw APIError.unknown("Could not get host from mutable request", "")
         }
 
-        request.setValue(URLRequestConstants.ContentType.applicationJson, forHTTPHeaderField: URLRequestConstants.Header.contentType)
```

Comment:
This wouldn't pose a problem because we've set it up during the construction of [the URLRequest](https://github.com/aws-amplify/amplify-swift/pull/3190/files#diff-723cc1c8e740c666ddf72add879d38a32dc87851fb68147df7c807cceb7afb6dR61).